samples/csharp/getting-started/DatabaseIntegration/DatabaseIntegration/DatabaseIntegration.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="3.0.0-preview.19074.3" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="3.0.0-preview.19074.3" />
+    <PackageReference Include="Microsoft.ML" Version="$(MicrosoftMLVersion)" />
```

Comment:
How is MicrosoftMLVersion set? Does this prevent anyone from just cloning and running the project as normal? I suspect either having fixed dependencies and updating the projects to use the latest, or using '1.0.0*'s would probably be better.

Reply:
This is a great question. I was following what other projects used,  thinking this was defined in a central place (i.e. I was hoping it was a props file that we include), however when looking for where this is defined, I found this:
[https://github.com/dotnet/machinelearning-samples/blob/e78e227d0269accbc74db26a1edffd0b782d2511/samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/Directory.Build.props#L4](https://github.com/dotnet/machinelearning-samples/blob/e78e227d0269accbc74db26a1edffd0b782d2511/samples/csharp/getting-started/DeepLearning_ObjectDetection_Onnx/Directory.Build.props#L4)

I dont understand how this is exactly working. I am setting to 1.0.0-preview for now and will likely file an issue on this. 



---
In reply to: [278285209](https://github.com/dotnet/machinelearning-samples/pull/295#discussion_r278285209) [](ancestors = 278285209)